### PR TITLE
[Fortinet] Separate Fortinet FortiManager into own Integration

### PR DESCRIPTION
**Pull request by legoguy1000**
## What does this PR do?

Separate Fortinet FortiManager into own Integration

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- Relates #1936

### CODE REVIEW

**Comment by legoguy1000** [comment]
@andrewkroh ready for testing

**Comment by legoguy1000** [comment]
I found and fixed the issues

File: packages/fortinet_fortimanager/data_stream/log/_dev/test/pipeline/test-common-config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+dynamic_fields:
+  event.ingested: ".*"
```
Comment (efd6) [inline packages/fortinet_fortimanager/data_stream/log/_dev/test/pipeline/test-common-config.yml:2]: This should not be needed.

Comment (efd6) [inline packages/fortinet_fortimanager/data_stream/log/_dev/test/pipeline/test-common-config.yml:2]: Delete.

File: packages/fortinet_fortimanager/data_stream/log/agent/stream/log.yml.hbs [hunk 2]
```diff
@@ -0,0 +1,3100 @@
+paths:
```
Comment (efd6) [inline packages/fortinet_fortimanager/data_stream/log/agent/stream/log.yml.hbs:1]: I take it whitespace differences in this file are due to editor settings?

Comment (legoguy1000) [reply packages/fortinet_fortimanager/data_stream/log/agent/stream/log.yml.hbs:1]: what specifically? The file should be the same as the original.

Comment (efd6) [reply packages/fortinet_fortimanager/data_stream/log/agent/stream/log.yml.hbs:1]: Yeah there are changes in mixed indent (some tabs have changed to spaces (it is now more consistent). Lines in the range 2550-3056 in the original have tabs, this has 2-space.

File: packages/fortinet_fortimanager/data_stream/log/_dev/test/pipeline/test-common-config.yml (hunk 1 above)
Comment (efd6) [reply packages/fortinet_fortimanager/data_stream/log/_dev/test/pipeline/test-common-config.yml]: This one is still there.

Comment (efd6) [reply packages/fortinet_fortimanager/data_stream/log/_dev/test/pipeline/test-common-config.yml]: No it's not. Sorry.

File: packages/fortinet_fortimanager/data_stream/log/agent/stream/log.yml.hbs (hunk 2 above)
Comment (legoguy1000) [reply packages/fortinet_fortimanager/data_stream/log/agent/stream/log.yml.hbs:1]: ahh ya pretty sure my VScode uses spaces for tabs.

**Review by efd6 — APPROVED** [review]

**Pull request merged**